src/scripts/Controller.js

```diff
@@ -518,6 +521,25 @@ export default class SitemapController {
 			},
 		});
 	}
+	readBlob() {
+		let files = document.getElementById('sitemapFiles').files;
```

Comment:
We consider using Jquery, so can you do it in jquery style. 
`$("#sitemapFiles")[0].files `